src/platforms/hosted/ftdi_swd.c

```diff
@@ -75,6 +74,15 @@ bool ftdi_swd_possible(void)
 	return true;
 }
 
+#if defined(_MSC_VER) && !defined(__clang__)
+static inline uint32_t __builtin_ctz(uint32_t i)
```

Comment:
`value` instead of `i`? We'd like not to introduce a new instance of the clang-tidy lint for too short a variable name. Please initialise `result` to 0 for use-before-init safety.